docs/backup_and_restore.rst

```diff
@@ -153,8 +153,6 @@ Once the restore is done, the Application Server will use the original Source an
 *Journalist Interface* Onion URLs. You will need to update the corresponding
 files on the Admin Workstation:
 
-.. todo:: We really should automate this process for Admins.
```

Comment:
imho fine to remove, this is no longer a priority to implement as: 

1. the correct behavior is documented for admins, 
2. the restore part of the backup/restore is a rare action. as such, engineering effort spent on automating it does not produce benefits to end users as much as automating more common tasks e.g. the journalist provisioning process (issue #1177, a common request by admins)